src/index.js

```diff
@@ -64,33 +64,35 @@ const userCollection = "partners";
 const onlyLoadWhenReady = true;
 // Create Redux data-store and store it in store and apply thunk middleware
 const store = createStore(
-  rootReducer,
-  compose(
-    applyMiddleware(
-      thunk.withExtraArgument({
-        getFirebase, // Firebase
-        getFirestore, // Cloud Firestore Database
-      })
-    ),
-    reduxFirestore(fbInit),
-    reactReduxFirebase(fbInit, {
-      useFirestoreForProfile: syncUserToAuth,
-      userProfile: userCollection,
-      attachAuthIsReady: onlyLoadWhenReady,
-    })
-  )
+	rootReducer,
```

Comment:
Wrong tab size?

Reply:
**Calling it tab size is missleading.**
It should not contain tabs, but spaces.
This issue has been fixed, by converting all indenty to spaces. My laptop was still using tabs instead of spaces.